test/api/core_test.dart

```diff
@@ -153,9 +154,13 @@ void main() {
           ..which(condition));
     }
 
-    checkRequest(http.ClientException('Oops'), it()..message.equals('Oops'));
-    checkRequest(const TlsException('Oops'), it()..message.equals('Oops'));
-    checkRequest((foo: 'bar'), it()..message.equals('Network request failed'));
+    final zulipLocalizations = lookupZulipLocalizations(ZulipLocalizations.supportedLocales.first);
+    checkRequest(http.ClientException('Oops'), it()..message(zulipLocalizations).equals('Oops'));
+    checkRequest(const TlsException('Oops'), it()..message(zulipLocalizations).equals('Oops'));
+    final expectedMessage = zulipLocalizations.apiConnectionNetworkRequestFailed;
+    checkRequest((foo: 'bar'), it()
+      ..message(zulipLocalizations)
+      .equals(expectedMessage));
```

Comment:
This bit seems cleanest to read if `expectedMessage` is inlined. Shorter; more parallel to the other examples just above; and the last line gets longer, but it's under 80 columns so that in itself is no problem at all.